src/coreComponents/constitutive/unitTests/testTriaxial_elasticIsotropic.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" ?>
+
+<Problem>
+  <Tasks>
+    <TriaxialDriver
+      name="triaxialDriver"
+      material="elastic"
+      mode="triaxial" 
+      strainFunction="strainFunction"
+      stressFunction="stressFunction"
+      steps="50" 
```

Comment:
Loadings are defined by stress/strain functions? Can we define also pore pressure (for poroelastic coupling test), changing stress path, etc?

Reply:
Yes, maybe in a follow up PR?  I'm keeping a list of additional features, and this would be a good one.

Reply:
Along the same line, a "wellbore driver" could be developed to solve the operational problems, which could be calibrated with both analytical solutions and experimental/field data.

Reply:
@jhuang2601, I'll add this to the feature list.  We will need to talk about exactly what this looks like though.